modules/serverless-logic-linking-workflows-eventing-system.adoc

```diff
@@ -0,0 +1,129 @@
+
+The `example-broker` is a Kafka class Broker, and its configuration is defined in the `kafka-broker-config` ConfigMap.
+
+The following example displays how to configure a Kafka Knative Broker for use with the SonataFlowPlatform:
+
+.Example of Knative Kafka Broker used by the SonataFlowPlatform
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: Kafka <1>
+  name: example-broker
+  namespace: example-namespace
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing
+----
+
+<1> The Kafka class is used to create the `example-broker`.
+
+The following example displays how the `example-workflow` is automatically linked to the `example-broker` in the `example-namespace` for event production and consumption:
+
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  name: example-workflow
+  namespace: example-namespace
+  annotations:
+    sonataflow.org/description: Example Workflow
+    sonataflow.org/version: 0.0.1
+    sonataflow.org/profile: preview
+spec:
+  flow:
+    start: ExampleStartState
+    events:
+      - name: outEvent1
+        source: ''
+        kind: produced
+        type: out-event-type1 <1>
+      - name: inEvent1
+        source: ''
+        kind: consumed
+        type: in-event-type1 <2>
+      - name: inEvent2
+        source: ''
+        kind: consumed
+        type: in-event-type2 <3>
+    states:
+      - name: ExampleStartState
+    ...
+----
+
+<1> The `example-workflow` outgoing events are produced by using the SinkBinding `example-workflow-sb`.
+<2> Events of type `in-event-type1` are consumed using the `example-workflow-inevent1-b40c067c-595b-4913-81a4-c8efa980bc11` trigger.
+<3> Events of type `in-event-type2` are consumed using the `example-workflow-inevent2-b40c067c-595b-4913-81a4-c8efa980bc11` trigger.
+
+You can list the automatically created SinkBinding `example-workflow-sb` by using the following command:
+
+.Example of Knative Eventing SinkBinding listing
+[source,terminal]
+----
+$ oc get sinkbindings -n example-namespace
+----
+
+.Example output
+[source,terminal]
+----
+NAME                                          TYPE            RESOURCE                                   SINK                              READY
+example-workflow-sb                  SinkBinding   sinkbindings.sources.knative.dev   broker:example-broker   True
+----
+
+You can use the following command to list the automatically created triggers for event consumption:
+
+.Example of Knative Eventing trigger listing
+[source,terminal]
+----
+$ oc get triggers -n example-namespace
+----
+
+.Example output
+[source,terminal]
+----
+NAME                                                              BROKER           SINK                                                     AGE   CONDITIONS   READY   REASON
```

Comment:
In the preview the formatting of headings to output is off. Can this be corrected?

Maybe the Verbatim attribute might help solidify spaces.

![Screenshot from 2025-01-08 12-45-19](https://github.com/user-attachments/assets/99fca6a4-4544-48ef-a73b-ef8149cbf51c)